install.sh

```diff
@@ -23,9 +23,9 @@ function display_help()
 # #################################################
 # global variables
 # #################################################
-default_path=/opt/rocm
+ROCM_PATH=$ROCM_PATH
 build_package=false
-install_prefix=$default_path
+install_prefix=$ROCM_PATH
```

Comment:
install_prefix=${ROCM_PATH:="/opt/rocm"}

Reply:
Thanks. Just renamed  "install_prefix" to "ROCM_PATH". It's working well now.